wcfsetup/install/files/lib/system/html/output/HtmlOutputProcessor.class.php

```diff
@@ -113,4 +119,22 @@ protected function getHtmlOutputNodeProcessor() {
 		
 		return $this->htmlOutputNodeProcessor;
 	}
+	
+	/**
+	 * Enables rel=ugc for external links.
+	 * 
+	 * @param bool $enable
+	 */
+	public function enableUgc($enable = true) {
```

Comment:
```suggestion
	public function setUgc($enable = true) {
```

`enableUgc` does not sound like the method would take a parameter, but instead always enable it (requiring you to use `disableUgc`).